src/main/scala/examples/Router.scala

```diff
@@ -32,7 +32,7 @@ class Packet extends Bundle {
   *
   * @param n is the number of fanned outputs for the routed packet
   */
-class RouterIO(n: Int) extends Bundle {
+class RouterIO(val n: Int) extends Bundle {
```

Comment:
I'm curious as to why this became a val, I can't see a reference to it.

Reply:
Auto clone type (rightly) complains about bundle parameters that aren't immutable.

Reply:
Got it, thanks.